Nos/Models/Nos.xcdatamodeld/Nos.xcdatamodel/contents

```diff
@@ -8,13 +8,20 @@
         <attribute name="profilePhotoURL" optional="YES" attributeType="URI"/>
         <relationship name="events" optional="YES" toMany="YES" deletionRule="Nullify" destinationEntity="Event" inverseName="author" inverseEntity="Event"/>
     </entity>
+    <entity name="ETag" representedClassName="ETag" syncable="YES" codeGenerationType="category">
```

Comment:
Personally I would rather name this model something with more semantic meaning, because ETag doesn't mean much if you don't know how Nostr mentions work under the hood. What do you think of a name like "EventReference" or "EventMention" or something similar?

Reply:
I like `EventReference`, that makes a lot of sense